lightning/src/routing/router.rs

```diff
@@ -163,12 +164,19 @@ struct RouteGraphNode {
 	/// The effective htlc_minimum_msat at this hop. If a later hop on the path had a higher HTLC
 	/// minimum, we use it, plus the fees required at each earlier hop to meet it.
 	path_htlc_minimum_msat: u64,
+	/// All penalties incurred *after* this hop on the way to the destination, as calculated using
+	/// channel scoring.
+	next_hops_penalty_msat: u64,
```

Comment:
Use of `next_hops` in these confused me somewhat - `next_hops_fee_msat` (both in a `PathBuildingHop` and in the add_node macro) refers to only fee after the hop but not including the fee charged at the hop, whereas here we're using it to include the penalty charged at a hop. Maybe `path_penalty_msat` or so?

Reply:
Ah, right. I was a little confused by the naming, but now I see the pattern. Should be consistent now.